omitted unneeded content in abstracts

jc-berger opened the pull request:
Throughout our migration docs, we have a redundant way of saying "projects in v1 devfiles still work in v2 devfiles." I went through our migration docs and removed this unnecessary phrasing. We can continue to  improve expand upon our abstracts in later PRs.

tstastna reviewed (approved):
LGTM!

Is the information you're removing preserved somewhere, for example in some overview of the migration process? Or is it so obvious that we don't need to mention it? :)

jc-berger commented:
@tstastna good question :) The information I removed is unnecessary, so no need to preserve. _But_ there have definitely been other PRs where I did have to preserve content I removed, so good catch!
